pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -954,6 +954,9 @@ type OutpostInfo interface {
 	GetOutpost() *Outpost
 }
 
+// ErrUnsupportedLocalCluster is an error for when an unsupported operation is attempted on a local cluster.
+var ErrUnsupportedLocalCluster = errors.New("this operation is not supported on local clusters")
```

Comment:
Can we mention in the error that these are private clusters/outpost?

Reply:
I haven't changed the wording of the error message in this PR, but it makes sense to include Outposts in the error message.